ui/templates/welcome.html

```diff
@@ -29,3 +30,14 @@
         </div>
     {% endif %}
 {% endblock %}
+{% block postcontent %}
+{{ block.super }}
+{% compress js %}
+  <script type="text/javascript"
```

Comment:
@amir-qayyum-khan: Can you remove the code from this page and just add `require(["jquery", "bootstrap"])` to `base.html` right after the `require.config({...})` statement?